cves/CVE-2013-0863.yml

```diff
@@ -89,6 +91,7 @@ fixes_vcc_instructions: |
   The notes field is optional - place anything that clarifies things or you
   find interesting in there.
 fixes:
+- commit: 29e0357a11680842a6b0313f76e0ca43e1f91501
```

Comment:
Nice job finding both fixes! I'd add a short note explaining what each did, as one introduces size checks while another deals with pointers. Learning how each works can help to inform future readers.